morpheus/_lib/tests/CMakeLists.txt

```diff
@@ -38,10 +40,10 @@ add_executable(test_libmorpheus
 
 target_link_libraries(test_libmorpheus
   PRIVATE
-    GTest::gtest
-    matx::matx
-    morpheus
-    pybind11::embed
+  GTest::gtest
+  matx::matx
+  morpheus
+  pybind11::embed
```

Comment:
Undo this indentation. There is a setting that controls whether or not we have arguments indented.